README.md

```diff
@@ -71,8 +71,9 @@ Use the `jaribu` executable in order to test single files, like so e.g.:
 
 ### How to build
```

Comment:
This is now part of the docs: https://remotestoragejs.readthedocs.io/en/latest/contributing/docs.html#how-to-build-the-docs-on-your-machine

We also have this issue open for adjusting the README: #1000

Also, wow, I didn't notice when I created issue number 1000. :)